package.json

```diff
@@ -68,12 +68,14 @@
     "react-dates": "18.5.0",
     "react-dom": "17.0.2",
     "react-dropzone": "6.2.4",
+    "react-error-boundary": "^3.1.4",
```

Comment:
📌

Reply:
totally, its better but not ideal, sounds like a plan